### ISSUE TITLE
Items

### ISSUE THREAD

**Issue by ShooshPapper** [taoneill/war#647]
This probably won't be a popular idea, but its' still an idea, so it's worth sharing.
Maybe we could do something where instead of having set kits for weapons, people can choose their own weapons for war matches? And if the zone owner doesn't want a certain weapon in their zone, there should be a plugin that disables the use of those weapons.
I prefer using bows.
Some others may prefer using swords.
So basically we get to choose our own war weapons?
